src/Events/Authorization/SubscriptionXacmlMapper.cs

```diff
@@ -0,0 +1,150 @@
+            }
+
+            if (!string.IsNullOrWhiteSpace(app))
+            {
+                resourceCategory.Attribute.Add(DecisionHelper.CreateXacmlJsonAttribute(AltinnXacmlUrns.AppId, app, DefaultType, DefaultIssuer));
+            }
+
+            resourceCategory.Attribute.Add(CreateResourceAttribute(subscriptionSubjectFilter));
+            resourceCategory.Attribute.Add(DecisionHelper.CreateXacmlJsonAttribute(AltinnXacmlUrns.AppResource, "events", DefaultType, DefaultIssuer));
+
+            return resourceCategory;
+        }
+
+        private static XacmlJsonCategory CreateSubjectCategory(string subscriptionConsumer)
+        {
+            XacmlJsonCategory subjectAttributes = new()
+            {
+                Attribute = CreateSubjectAttributes(subscriptionConsumer)
+            };
+
+            return subjectAttributes;
+        }
+
+        private static List<XacmlJsonAttribute> CreateSubjectAttributes(string consumer)
+        {
+            List<XacmlJsonAttribute> attributes = new();
+
+            if (consumer.StartsWith(UserPrefix))
+            {
+                string value = consumer.Replace(UserPrefix, string.Empty);
+                attributes.Add(DecisionHelper.CreateXacmlJsonAttribute(ClaimUserId, value, ClaimValueTypes.String, DefaultIssuer));
+            }
+            else if (consumer.StartsWith(OrgPrefix))
+            {
+                string value = consumer.Replace(OrgPrefix, string.Empty);
+                attributes.Add(DecisionHelper.CreateXacmlJsonAttribute(ClaimOrg, value, ClaimValueTypes.String, DefaultIssuer));
+            }
+            else if (consumer.StartsWith(PartyPrefix))
+            {
+                string value = consumer.Replace(PartyPrefix, string.Empty);
+                attributes.Add(DecisionHelper.CreateXacmlJsonAttribute(ClaimPartyID, value, ClaimValueTypes.Integer32, DefaultIssuer));
+            }
+
+            return attributes;
+        }
+
+        private static XacmlJsonAttribute CreateResourceAttribute(string resource)
+        {
+            if (resource.StartsWith(UserPrefix))
+            {
+                string value = resource.Replace(UserPrefix, string.Empty);
+                return DecisionHelper.CreateXacmlJsonAttribute(ClaimUserId, value, ClaimValueTypes.String, DefaultIssuer);
+            }
+            else if (resource.StartsWith(OrgPrefix))
+            {
+                string value = resource.Replace(OrgPrefix, string.Empty);
+                return DecisionHelper.CreateXacmlJsonAttribute(ClaimOrg, value, ClaimValueTypes.String, DefaultIssuer);
+            }
+            else if (resource.StartsWith(PartyPrefix))
+            {
+                string value = resource.Replace(PartyPrefix, string.Empty);
+                return DecisionHelper.CreateXacmlJsonAttribute(ClaimPartyID, value, ClaimValueTypes.Integer, DefaultIssuer);
```

Comment:
using interger ans not integer32, as ABAC failed when using int32 when testing locally. Ref https://github.com/Altinn/altinn-studio/blob/master/src/Altinn.Platform/Altinn.Platform.Authorization/Altinn.Authorization.ABAC/Utils/XacmlJsonXmlConverter.cs#L219 
Is this ok?